newsroom/wire/formatters/picture.py

```diff
@@ -27,7 +28,8 @@ def format_item(self, item, item_type="items"):
         if not picture:
             raise ValueError("Undefined picture")
 
-        picture_details = picture.get("renditions", {}).get("baseImage", {})
+        renditions = picture.get("renditions", {})
+        picture_details = renditions.get(app.config.get("DOWNLOAD_RENDITION")) or renditions.get("baseImage", {})
```

Comment:
would be good to check if `picture_details is None` and somehow handle it, otherwise this might raise an error